app/models/scheme.rb

```diff
@@ -295,6 +295,16 @@ def active?
     status == :active
   end
 
+  def has_active_locations?
+    active? && locations.active.exists?
```

Comment:
I'm just having a look at `active` scope, and I think it's doing something different from what I assumed at first. I think we should actually use `active_status` scope here.

If we look at `filter_by_status` scope, we're actually replacing `active` with `active_status`


This might need refactoring, but maybe separate from this PR (unless you're keen to look into where/how `active` scope is being used).